test/StakeManager.js

```diff
@@ -207,7 +206,7 @@ describe('StakeManager', function () {
       const stake = tokenAmount('20000');
       const stakerIdAcc1 = await stakeManager.stakerIds(signers[1].address);
       const stakeBeforeAcc1 = (await stakeManager.stakers(stakerIdAcc1)).stake;
-      await schellingCoin.connect(signers[1]).approve(stakeManager.address, stake);
+      await Coin.connect(signers[1]).approve(stakeManager.address, stake);
```

Comment:
?

Reply:
This I have changed last test case is not passing fixing it will commit soon.

Reply:
All errors will be fixed @hrishikeshio